### ISSUE TITLE
Licensing issue

### ISSUE THREAD

**Issue by jacobbudin** [aalderi/TiAndroidCanvas#1]
Hello, thanks for creating this module. I'm confused as to whether it needs to be licensed.

After implementing the `TiAndroidCanvas` module, I'm receiving these errors on starting the application:

```
[WARN]  TiVerify: (Timer-0) [1297,5005] Verifying module licenses...
[ERROR] TiVerify: (Timer-0) [762,5767] License violation detected. Unlicensed module(s) detected.  You must be a subscriber to use one or more of the modules included with this project.. Please contact Appcelerator Support.
```

Additionally I see these dialogs during runtime:

![](http://i.imgur.com/q4HlcDh.png)

Any assistance would be appreciated. Thanks.

**Comment by AppWerft** [comment]
I have just successful compiled the module and linked in my project. It works fine.
Perhaps you must change the guid in manifest. 

![](http://i.imgur.com/Z1BaGfW.png)

**Comment by AppWerft** [comment]
Now I have the same issue in an other project. You find a new module here: [https://github.com/AppWerft/DLRmediathek/modules/android/](https://github.com/AppWerft/DLRmediathek/modules/android/) It is compiled with other guid and in my project it runs without error message.

**Comment by jacobbudin** [comment]
I agree that will work, but I don't see why it should be necessary. If this is an open source project, the GUID shouldn't map to Appcelerator's Marketplace as a licensed add-on, which [seems to be the consensus about this issue](http://developer.appcelerator.com/question/125625/unlicensed-module) (here's [another explanation](https://developer.appcelerator.com/question/160130/unlicensed-modules-detected)). Is it possible you accidentally "reused" a GUID from a different project?
